content/1-reactivity/2-update-state/vue2/Name.vue

```diff
@@ -0,0 +1,16 @@
+<script>
+export default {
+	data() {
+		return {
+			name: 'John',
+		};
+	},
+	create() {
+		this.name = 'Jane';
+	},
+};
```

Comment:
The spacing is kind of big, is it done automatically with Prettier ?

Reply:
Yes, prettier --write will be executed after commit, and then the interval will be displayed on GitHub, but it is normal in my editor

Reply:
I found that if this useTabs is true, it is normal. Can I add "**/vue2/*" to the files in the overrides below?